public/javascripts/SVValidate/mobile/mobileValidate.js

```diff
@@ -21,6 +21,11 @@ $(document).ready(function() {
 
     resizeMobileValidation();
 
+    //add the 'animate-button-0' class to all validaton buttons so an animation is performed on click
```

Comment:
Can you just update the comment to conform to [the style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide)? Everything else looks great :)